teamwork/apps/courses/views/EditCourseView.py

```diff
@@ -74,7 +74,7 @@ def edit_course(request, slug):
         if added:
             messages.add_message(request, messages.SUCCESS, "Successfully added member(s) to course.")
         else:
-            messages.add_message(request, messages.SUCCESS, "Failed to successfully add member(s) to course.")
```

Comment:
Why was this message changed?

Reply:
It wasn't we were trying to fix that task so we were just playing around with it. But we fixed this task that you assigned for us on the PR.

Reply:
Apologies, this message is correct